### PR TITLE
Add Centos 7 walkthrough

### PR DESCRIPTION
**Pull request by manics**
Add a walkthrough for installing OMERO.server on Centos 7. See linux/test/centos7/README.md

TODO:
- [x] Add systemd files to start OMERO and OMERO.web
- [x] Either add `install-centos7.sh` to `install.sh` or split `install.sh` into one file per environment.

### CODE REVIEW

**Comment by joshmoore** [comment]
Tested everything up to https://github.com/manics/omero-install/commit/7c56c7df2efa6f51f357b41af05376f3b0ac13f4, i.e.

```
+        ./docker-build.sh centos7
+        CID=$(docker run -d ... -v /sys/fs/cgroup:/sys/fs/cgroup:ro omero_install_test_centos7)
+        ssh -o UserKnownHostsFile=/dev/null root@<address of container> # Password: omero
+        cd /omero-install-test
+        bash install-centos7.sh
+        echo omero:omero | chpasswd
```
works fine. If there's any simplification we can do for tomorrow, :thumbsup: Otherwise, we'll make do.

**Comment by jburel** [comment]
after adding line to install i.e. ``` bash install-centos7.sh``` and  run ```./docker-build.sh centos7```
same error I came up with my branch i.e. 
```
Failed to get D-Bus connection: No connection to service manager.
failed to find PGDATA setting in postgresql-9.4.service
```

File: linux/test/centos7/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Create a test image containing the installation scripts
+2. Start the container
+3. ssh in
+4. Change into the `/omero-install-test` directory
+5. Run the scripts
+6. Optionally set a system password for the `omero` user to allow ssh
+
+        ./docker-build.sh centos7
+        CID=$(docker run -d ... -v /sys/fs/cgroup:/sys/fs/cgroup:ro omero_install_test_centos7)
```
Comment (joshmoore) [inline linux/test/centos7/README.md]: from `docker-machine` with vbox, only `--privileged` works.

**Comment by manics** [comment]
@joshmoore README updated

**Comment by jburel** [comment]
After clarification in the doc, managed to ssh in.

* Started manually omero server and web. 
* Import an image and view it in both client web and insight.

**Comment by dominikl** [comment]
Neither of the option worked for me, the docker image apparently doesn't start up.

**Comment by dominikl** [comment]
Forget my last comment, it works! Had a typo, didn't spot it and used bash history all the time.

**Comment by manics** [comment]
systemd files added https://github.com/manics/omero-install/commit/caf572ac8a6bf7f07188bffd768450d8544c2fde /cc @stick @kennethgillen

**Comment by jburel** [comment]
retested with latest commits.
 I locally uncommented line 23/24 in ```install-centos7`` before building
Both omero server and omero web started w/o problems
able to import images, view them in both web and insight, run image export script.

**Comment by jburel** [comment]
apache support added https://github.com/manics/omero-install/pull/1

**Comment by joshmoore** [comment]
Merging all PRs so someone can open a cleaning/renaming PR.

**Comment by jburel** [comment]
Will close my PR against this one and re-open a new one for apache

**Pull request merged**